massgov/mayflower
# DP-18422 Org page featured message

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

Add a new template to accommodate a comp heading and rich text followed by a callout link in a container which is visually resembled to a callout link with the white theme as a featured message.

Place the featured message sample to org page right above the mosaic.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-18422)
- [Github issue]()

## Steps to Test


1. Go to the organization page.
2. Check the callout message matches the [spec](https://eotss.invisionapp.com/share/VUX085PCBG9#/screens/414936204_Overview-_Design_For_New_Section_On_Org_Page_-name_TBD-).

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
